packages/strapi-plugin-upload/admin/src/containers/HomePage/index.js

```diff
@@ -104,6 +110,11 @@ const HomePage = () => {
     ],
   };
 
+  const params = {
+    _limit: parseInt(getQueryValue('_limit'), 10) || 10,
+    _page: parseInt(getQueryValue('_page'), 10) || 1,
```

Comment:
This not a parameter that we support in order to do the pagination can you use the `_start`  one?

Here is [our documentation](https://strapi.io/documentation/3.0.0-beta.x/content-api/parameters.html#available-operators)